src/pages/Sign/sections/SignUp.tsx

```diff
@@ -121,79 +149,137 @@ const UserSchema = Yup.object().shape({
-              {(title) => title}
-            </TranslationsPlaceholderConsumer>
-          </Header>
-        </HeaderWrapper>
-        <Formik
-          initialValues={{
-            email: '',
-            personalNumber: '',
-          }}
-          validationSchema={UserSchema}
-          onSubmit={(values) => alert(JSON.stringify(values, null, 2))}
-        >
-          {({ errors, touched }) => (
-            <CustomForm>
-              <InputTitle>
-                <TranslationsConsumer textKey="SIGN_INPUT_ONE_TITLE">
-                  {(title) => title}
-                </TranslationsConsumer>
-              </InputTitle>
-
-              <InputField name="email" />
-              {errors.email && touched.email ? (
-                <ErrorMessage>
-                  <TranslationsConsumer textKey={errors.email}>
-                    {(errorMessage) => errorMessage}
-                  </TranslationsConsumer>
-                </ErrorMessage>
-              ) : (
-                <ErrorMessage />
-              )}
-              <InputTitle>
-                <TranslationsConsumer textKey="SIGN_INPUT_TWO_TITLE">
-                  {(title) => title}
-                </TranslationsConsumer>
-              </InputTitle>
-              <InputField name="personalNumber" />
-              {errors.personalNumber && touched.personalNumber ? (
-                <ErrorMessage>
-                  <TranslationsConsumer textKey={errors.personalNumber}>
-                    {(errorMessage) => errorMessage}
-                  </TranslationsConsumer>
-                </ErrorMessage>
-              ) : (
-                <ErrorMessage />
-              )}
-              <GetInsuredButton>
-                <TranslationsConsumer textKey="SIGN_BUTTON_TEXT">
-                  {(buttonText) => (
-                    <InputSubmit type="submit" value={buttonText} />
-                  )}
-                </TranslationsConsumer>
-              </GetInsuredButton>
-              <TranslationsConsumer textKey="SIGN_BANKID_ERROR_MESSAGE">
-                {(errorMessage) =>
-                  errorMessage ? <ErrorText>{errorMessage}</ErrorText> : null
-                }
-              </TranslationsConsumer>
-            </CustomForm>
-          )}
-        </Formik>
-      </Card>
-    </CardWrapper>
-  </OuterWrapper>
+  <SessionTokenGuard>
+    <OuterWrapper>
+      <CardWrapper>
+        <Card>
+          <HeaderWrapper>
+            <Header>
+              <OfferContainer>
+                {(offer) => (
+                  <TranslationsPlaceholderConsumer
+                    textKey="SIGN_HEADER_TITLE"
+                    replacements={{
+                      address: offer.insurance.address,
+                    }}
+                  >
+                    {(title) => title}
+                  </TranslationsPlaceholderConsumer>
+                )}
+              </OfferContainer>
+            </Header>
+          </HeaderWrapper>
+          <SessionContainer>
```

Comment:
We should remove this `<SessionContainer />` as it no longer adds any value, given our `<SessionTokenGuard />`at the top.

```suggestion

```